### PR TITLE
R4_HealthConcern_Support

### PR DESCRIPTION
**Pull request by UjwalaBheema**
Description
----
Health Concern support added for Condition category
Patient Access support for Condition

PR Checklist
----
- [x] Screenshot(s) of changes attached before changes merged.
Screenshot 2020-12-10 at 5 15 02 PM
Screenshot 2020-12-10 at 5 15 18 PM
Screenshot 2020-12-10 at 5 15 34 PM
Screenshot 2020-12-10 at 5 15 51 PM
Screenshot 2020-12-10 at 5 16 02 PM
Screenshot 2020-12-10 at 5 16 14 PM
Screenshot 2020-12-10 at 5 16 32 PM
Screenshot 2020-12-10 at 5 16 40 PM

- [ ] Screenshot(s) of changes attached after changes merged and published.

### CODE REVIEW

**Review by aki-mathur — COMMENTED** [review]
Can we add patient access examples along with Condition.
Reference: https://github.com/cerner/fhir.cerner.com/pull/505

File: content/millennium/r4/general-clinical/condition.md [hunk 1]
```diff
@@ -25,6 +25,7 @@ The following fields are returned if valued:
 * [Resolved date  (only applies to problems)](https://hl7.org/fhir/R4/condition-definitions.html#Condition.abatement_x_)
```
Comment (ashu-bdvt) [inline content/millennium/r4/general-clinical/condition.md:25]: Update as 'Resolved date (only applies to problems and health concerns)' and remove additional space between date and (

File: content/millennium/r4/general-clinical/condition.md [hunk 2]
```diff
@@ -25,6 +25,7 @@ The following fields are returned if valued:
...
 * [Date recorded](https://hl7.org/fhir/R4/condition-definitions.html#Condition.recordedDate)
 * [Who recorded the condition](https://hl7.org/fhir/R4/condition-definitions.html#Condition.recorder)
+* [Asserter](http://hl7.org/fhir/r4/condition-definitions.html#Condition.asserter)
```
Comment (ashu-bdvt) [inline content/millennium/r4/general-clinical/condition.md:28]: Add {:target="_blank}

File: content/millennium/r4/general-clinical/condition.md [hunk 3]
```diff
@@ -39,11 +40,11 @@ Search for Conditions that meet supplied query parameters:
 
 _Implementation Notes_
 
-* Currently `problem-list-item` and `encounter-diagnosis` are supported.
+* Currently `problem-list-item` `encounter-diagnosis` and `health-concern` are supported.
```
Comment (ashu-bdvt) [inline content/millennium/r4/general-clinical/condition.md:43]: add , between categories

**Review by ashu-bdvt — CHANGES_REQUESTED** [review]
Remove screenshots from fhir.cerner.com. Just add local screenshots with the changes

File: content/millennium/r4/general-clinical/condition.md [hunk 4]
```diff
@@ -53,7 +54,7 @@ _Implementation Notes_
  `patient`         | This or `_id` or `subject`     | [`reference`] | Who the condition is for. Example: `12345`
  `subject`         | This or `_id` or `patient`     | [`reference`] | Who the condition is for. Example: `Patient/12345`
  `clinical-status` | No                             | [`token`]     | The clinical status of the condition. Example: `active`, `inactive`, `resolved`
- `category`        | No                             | [`token`]     | The category of the condition. Categories problem-list-item and encounter-diagnosis are supported as of now. Example: `problem-list-item`, `encounter-diagnosis`
+ `category`        | No                             | [`token`]     | The category of the condition. Categories problem-list-item encounter-diagnosis and health-concern are supported as of now. Example: `problem-list-item`, `encounter-diagnosis`, `health-concern`
```
Comment (aki-mathur) [inline content/millennium/r4/general-clinical/condition.md:57]: Can you check the description with functional Jira

File: content/millennium/r4/general-clinical/condition.md [hunk 5]
```diff
@@ -122,6 +167,7 @@ _Implementation Notes_
 * Only the body fields mentioned below are supported. Unsupported fields will be ignored.
 * Modifier fields should not be provided, and will cause the transaction to fail.
 * The code.coding field can have at most 2 codings, one of which must be set as userSelected true and the other one must be set as userSelected false.
+* Currently `problem-list-item` and `encounter-diagnosis` are supported.
```
Comment (aki-mathur) [inline content/millennium/r4/general-clinical/condition.md:170]: This line should be added in Search not in Create

File: lib/resources/example_json/r4_examples_capabilitystatement.rb [hunk 6]
```diff
@@ -234,8 +234,8 @@ module Resources
                   'name': 'category',
                   'definition': 'http://hl7.org/fhir/R4/SearchParameter/Condition-category',
                   'type': 'token',
-                  'documentation': 'The category of the condition. Categories problem-list-item and '\
-                                   'encounter-diagnosis are supported as of now.'
+                  'documentation': 'The category of the condition. Categories problem-list-item '\
```
Comment (aki-mathur) [inline lib/resources/example_json/r4_examples_capabilitystatement.rb:237]: add , between the categories

**Review by aki-mathur — CHANGES_REQUESTED** [review]

File: content/millennium/r4/general-clinical/condition.md (hunk 4 above)
Comment (UjwalaBheema) [reply content/millennium/r4/general-clinical/condition.md:57]: Looks like in functional overview it was missed, its fine to add the supported categories

File: content/millennium/r4/general-clinical/condition.md (hunk 5 above)
Comment (UjwalaBheema) [reply content/millennium/r4/general-clinical/condition.md:170]: This is an extra line  for create currently, retrieve and search has three categories supported but create and update supoorts only two 
Update has the same line prior itself, just added the same here
Screenshot 2020-12-10 at 4 22 31 PM

File: lib/resources/example_json/r4_examples_capabilitystatement.rb (hunk 6 above)
Comment (UjwalaBheema) [reply lib/resources/example_json/r4_examples_capabilitystatement.rb]: Addressed

File: content/millennium/r4/general-clinical/condition.md (hunk 4 above)
Comment (aki-mathur) [reply content/millennium/r4/general-clinical/condition.md]: nit: can we add , between the categories

Comment (UjwalaBheema) [reply content/millennium/r4/general-clinical/condition.md]: Addressed

File: content/millennium/r4/general-clinical/condition.md (hunk 1 above)
Comment (UjwalaBheema) [reply content/millennium/r4/general-clinical/condition.md]: Addressed

File: content/millennium/r4/general-clinical/condition.md (hunk 2 above)
Comment (UjwalaBheema) [reply content/millennium/r4/general-clinical/condition.md:28]: View wont look properly, other links are not added _target blank in the same page

**Review by vinuthnag — APPROVED** [review]

**Review by aki-mathur — APPROVED** [review]

**Review by ashu-bdvt — APPROVED** [review]

**Pull request merged**